### ISSUE TITLE
Check if the KLEE-2.3 directory actually necessary to add here

### ISSUE THREAD

**Issue by sampollard** [sandialabs/xstack-klokkos#1]
When preparing for open-source release there's the klee-2.3 part (in the GaTech internal repo). I see a few files (like Shyamali's readme) that are definitely different, but can we confirm that we don't need the whole klee-2.3 repo, and can just point to a particular version? If not, we have permission to open-source our development branches of KLEE as well, so that may be a better place for the changes.

**Comment by vlkale** [comment]
I don't think we need to get rid of KLEE-23 directory, at least not immediately. I do think open-sourcing our development branches of Klee from X-stack would be good.

**Comment by sampollard** [comment]
For what it's worth, we got permission for the whole project, not just the KLOKKOS work. So it's possible to fork from the KLEE repos directly on Github, or we could copy the tree in here.

**Comment by sampollard** [comment]
Assigning this to you @vlkale to make a sub-module here for KLEE-2.3.

**Comment by sampollard** [comment]
After a discussion with Rick, it seems simplest to just add the snapshot of klee to this repo directly. Once @naegling gets access he'll ad that from Gatech.
